pkg/noderesourcetopology/filter.go

```diff
@@ -68,28 +68,51 @@ func resourcesAvailableInAnyNUMANodes(logKey string, numaNodes NUMANodeList, res
 
 	// Node() != nil already verified in Filter(), which is the only public entry point
 	nodeName := nodeInfo.Node().Name
+	nodeResources := util.ResourceList(nodeInfo.Allocatable)
 
 	for resource, quantity := range resources {
+		if quantity.IsZero() {
```

Comment:
How a resource request can be zero in the first place?
I don't see why this check is necessary.

Reply:
this check was present in the original code and just moved around. We can probably remove it entirely if we deem it unnecessary.

Reply:
still TBD. I'm not completely sure we want to remove this, event though it should not happen